pkg/registry/rbac/reconciliation/reconcile_role_test.go

```diff
@@ -268,8 +272,102 @@ func TestComputeReconciledRole(t *testing.T) {
 			t.Errorf("%s: Expected\n\t%v\ngot\n\t%v", k, tc.expectedReconciliationNeeded, reconciliationNeeded)
 			continue
 		}
+		result.Role.(ClusterRoleRuleOwner).ClusterRole.AggregationRule = nil
```

Comment:
why was this change needed? I would have expected it to be nil already

Reply:
> why was this change needed? I would have expected it to be nil already

removing.